lib/supa/commands/attribute.rb

```diff
@@ -2,8 +2,20 @@ module Supa
   module Commands
     class Attribute < Supa::Command
       def represent
+        return if hide?
+
         @tree[@name] = value
       end
+
+      private
+
+      def hide?
+        value.nil? && hide_when_nil?
+      end
+
+      def hide_when_nil?
```

Comment:
can we use the same logic behind name `hide_when_empty`? We use the same logic as ActiveSupport it's `empty/blank`, when it's empty in terms of object or `nil`